plugins/net_plugin/include/eosio/net_plugin/protocol.hpp

```diff
@@ -16,9 +16,10 @@ namespace eosio {
 
    // Longest domain name is 253 characters according to wikipedia.
    // Addresses include ":port" where max port is 65535, which adds 6 chars.
+   // Addresses may also include ":bitrate" with suffix and separators, which adds 30 chars.
```

Comment:
Can you explain why 30 characters.

Reply:
`:1,000,000,000,000` is 18 characters plus up to 5 characters of suffix, plus padding for future expansion.  I wanted to leave room for someone to configure every significant digit of a terabyte per second.  As implemented right now, throttling is very precise, never overshooting, and seldom undershooting by more than 12 bytes per second on a 1000 byte per second test limit.  A sensible person wouldn't write such a config, but a machine-generated one might.

Despite its location, that length is not part of the protocol.  The throttle bytes are always parsed off of the listen address specification before it is used on the network (if p2p-server-address is unspecified).

Reply:
Please add this to the comment.

Reply:
Added.

Reply:
I meant in the code.